website/docs/plugins/gradle-plugin-tasks.mdx

```diff
@@ -263,6 +289,7 @@ resulting generated code will be automatically added to the project main source
 | `serializer` | GraphQLSerializer | | JSON serializer that will be used to generate the data classes.<br/>**Default value is:** `GraphQLSerializer.JACKSON`. |
 | `schemaFile` | File | yes | GraphQL schema file that will be used to generate client code. |
 | `useOptionalInputWrapper` | Boolean | | Boolean opt-in flag to wrap nullable arguments in `OptionalInput` that distinguish between `null` and undefined/omitted value.<br/>**Default value is:** `false`.<br/>**Command line property is**: `useOptionalInputWrapper` |
+| `parserOptions` | GraphQLParserOptions | | Configures the settings used when parsing GraphQL queries and schema definition language documents. |
```

Comment:
let's keep the alphabetical order -> can you also fix `serializer` <-> `schemaFile` order?

*this comment applies to all doc changes

Reply:
should we also specify somewhere in the docs what are the available options?

Reply:
Fixed the ordering and added the specific options and defaults in the docs.